sdk/servicebus/Azure.Messaging.ServiceBus/samples/Sample01_HelloWorld.md

```diff
@@ -45,27 +45,27 @@ IList<ServiceBusMessage> messages = new List<ServiceBusMessage>();
 messages.Add(new ServiceBusMessage(Encoding.UTF8.GetBytes("First")));
 messages.Add(new ServiceBusMessage(Encoding.UTF8.GetBytes("Second")));
 // send the messages
-await sender.SendAsync(messages);
+await sender.SendMessagesAsync(messages);
 ```
 The second way of doing this is using safe-batching. With safe-batching, you can create a `ServiceBusMessageBatch` object,
 which will allow you to attempt to messages one at a time to the batch using TryAdd. If the message cannot fit in the batch,
 TryAdd will return false.
 
 ```C# Snippet:ServiceBusSendAndReceiveSafeBatch
-ServiceBusMessageBatch messageBatch = await sender.CreateBatchAsync();
+ServiceBusMessageBatch messageBatch = await sender.CreateMessageBatchAsync();
 messageBatch.TryAdd(new ServiceBusMessage(Encoding.UTF8.GetBytes("First")));
 messageBatch.TryAdd(new ServiceBusMessage(Encoding.UTF8.GetBytes("Second")));
```

Comment:
Should `TryAdd` be `TryAddMessage` instead?

Reply:
Hmm, perhaps. You are already in the context of a message batch, but this might make it even more clear, as well as make it stand out to users when looking through intellisense.

Reply:
I like this idea, but let's see what others think.

Reply:
/cc @jsquire

Reply:
I don't believe that the `Message` suffix really adds any value here, personally.  Maybe I'm overlooking something.  Is there a reason that you believe the simpler name would cause confusion that having a `Message` suffix would make clear?

**My Rationale**
- You're using a type that represents a batch of messages and does not hold other items.  
- You have a single `TryAdd` method that takes a single concrete type, explicitly `ServiceBusMessage` and a parameter named `message`.
- Intellisense prompts will clearly indicate the type and context, given that there is no generic signature that would hide the type and the parameter name is explicit.

Reply:
In UX studies, I noticed that users tend to overlook methods which seem like they might be base methods that don't apply to the task at hand. In particular, `TryAdd` didn't jump out to users, and they would have to read the docs to understand how to add a message to the batch.

Reply:
Personally, I don't think `TryAdd` is unclear. The reason I suggested it was because the changes in this PR are making method that work with `ServiceBusMessage` consistent by having the word `Message` in the method name. So, for that endeavor, I *seems* like it makes sense (for consistency) to have the word `Message` here. I know it is a different context (not Sender/Receiver/Processor, but for MessageBatch), but this isn't that far out of that space to follow the same pattern.

Reply:
That all makes sense. Your comment reminded me of UX study participants struggles with this type and method, so I think the change is a good one.